packages/log/log.go

```diff
@@ -73,9 +78,55 @@ func Fatalf(format string, args ...interface{}) {
 	os.Exit(1)
 }
 
+func DefaultJSONFormatter(i interface{}) ([]byte, error) {
+	return json.MarshalIndent(i, "", " ")
```

Comment:
I would separate by two white spaces for better readability.

```suggestion
	return json.MarshalIndent(i, "", "  ")
```